app/src/main/java/com/example/benchproject/app/common/binding/textView/TextViewBindingAdapters.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.benchproject.app.common.binding.textView
+
+import android.widget.TextView
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("addBudget")
+fun TextView.addBudgetToTextView(budget: Int) {
+    text = if (budget != 0) "$budget $" else "-"
+}
```

Comment:
Do we really need a binding adapter here? You can just expose an additional property or function from `responseMovieDetails`, eg `responseMovieDetails.budgetText`

Reply:
Generally try to avoid a binding adapters which can be used only for a specific, it will be even easier to test such logic in scope of a single object (like data class) instead of data binding adapter.

Reply:
Ok, I understand and I changed it